### PR TITLE
Add accelerate operator.

### PR DESCRIPTION
**Pull request by smimram**
Useful for testing by speeding up sources by dropping frames.

### CODE REVIEW

**Comment by smimram** [comment]
Can I integrate such an operator? I found it quite useful to debug transitions...

File: src/operators/accelerate.ml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    val mutable skipped = 0
+
+    (** Frame used for dropping samples. *)
+    val null = Frame.create kind
+
+    method private get_frame frame =
+      let pos = ref 1 in
+      (* Drop frames if we are late. *)
+      (* TODO: we could also duplicate if we are in advance. *)
+      while !pos > 0 && float (filled + skipped) /. float filled < ratio () do
```
[the remark this answers is not in the document]
Comment (toots) [reply src/operators/accelerate.ml:71]: For a naive randomized implementation, I could think of the following:
```ocaml
val full_cycle = [`Skip, `Skip, `Skip`, `Fill, `Fill, ...]
val mutable current_cycle = []

method private pick_next_operation =
  let randomize l =
    let nd = List.map (fun c -> (Random.bits (), c)) d in
    let sond = List.sort compare nd in
    List.map snd sond
  in
  match current_cycle with
     | [] ->
         current_cycle <- randomize current_cycle;
         List.hd current_cycle
     | op::tl ->
         current_cycle <- tl;
         op

File: src/operators/accelerate.ml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        Frame.clear null;
+        source#get null;
+        self#slave_tick;
+        pos := Frame.position null;
```
[the remark this answers is not in the document]
Comment (smimram) [reply src/operators/accelerate.ml:75]: How could it? It is a position in a frame!

File: src/operators/accelerate.ml (hunk 1 above)
[the remark this answers is not in the document]
Comment (smimram) [reply src/operators/accelerate.ml]: The implementation is now randomized by default. What do you think of it?

**Pull request merged**